src/plugins/guided_onboarding/public/constants/guides_config/observability.ts

```diff
@@ -59,13 +59,13 @@ export const observabilityConfig: GuideConfig = {
         title: i18n.translate(
           'guidedOnboarding.observabilityGuide.viewDashboardStep.manualCompletionPopoverTitle',
           {
-            defaultMessage: 'Explore the pre-built Kubernetes dashboards',
+            defaultMessage: 'Explore Kubernetes dashboards',
           }
         ),
         description: i18n.translate(
           'guidedOnboarding.observabilityGuide.viewDashboardStep.manualCompletionPopoverDescription',
           {
-            defaultMessage: `Take your time to explore out-of-the-box dashboards that are included with the Kubernetes integration. When you're ready, you can access the next step of the guide in the button above.`,
+            defaultMessage: `Take your time to explore these pre-built dashboards included with the Kubernetes integration. When you’re ready, click the Setup guide button to continue.`,
```

Comment:
> click the Setup guide button to continue

@kellyemurphy I think your recommendation was to prompt the user to click a "Next" button. However, that is not supported in the current design. Let me know what you think.

Reply:
Gotcha, it's ok as-is then.